Sources/SwiftDocC/LinkTargets/LinkDestinationSummary.swift

```diff
@@ -316,9 +320,87 @@ extension LinkDestinationSummary {
             return foundLanguage
         })
         platforms = try container.decodeIfPresent([AvailabilityRenderItem].self, forKey: .platforms)
-        taskGroups = try container.decodeIfPresent([TaskGroup].self, forKey: .taskGroups)
-        usr = try container.decodeIfPresent(String.self, forKey: .usr)
-        declarationFragments = try container.decodeIfPresent(DeclarationFragments.self, forKey: .declarationFragments)
         redirects = try container.decodeIfPresent([URL].self, forKey: .redirects)
+        
+        do {
+            let contentVariants = try container.decode([ContentVariant].self, forKey: .contentVariants)
+            guard !contentVariants.isEmpty else {
+                throw DecodingError.dataCorruptedError(forKey: .contentVariants, in: container, debugDescription: "Missing required content. ContentVariations is empty.")
+            }
+            self.contentVariants = contentVariants
+        } catch DecodingError.keyNotFound(_, let originalErrorContext) {
```

Comment:
Should we do a version check instead (`== "0.1.0"`) for the legacy decoding rather than falling back to it if the variants key is missing, or do you think it's not worth it?

Reply:
We don't encode the version information in the JSON.